editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx

```diff
@@ -640,6 +640,218 @@ describe('Select Mode Double Clicking', () => {
+    // prettier-ignore
+    const desiredPath = EP.fromString(
+      'sb' +                 // Skipped as it's the storyboard
+      '/scene-CardList' +    // Skipped because we skip over Scenes
+      '/CardList-instance' + // <- First double click
+      ':CardList-Col' +      // <- Third double click
+      '/CardList-Card~~~1' + // <- Fourth *and* Fifth double click, as the Fifth is required to focus it
+      ':Card-Root',          // <- Sixth double click
+    )
+
+    const renderResult = await renderTestEditorWithCode(
+      TestProjectAlpineClimbWithFragments,
+      'await-first-dom-report',
+    )
+
+    const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
+
+    const cardSceneRoot = renderResult.renderedDOM.getByTestId('generated-card-1')
+    const cardSceneRootBounds = cardSceneRoot.getBoundingClientRect()
+
+    const fireDoubleClickEvents = createDoubleClicker()
+
+    const doubleClick = async () => {
+      await act(async () => {
+        fireDoubleClickEvents(
+          canvasControlsLayer,
+          cardSceneRootBounds.left + 130,
+          cardSceneRootBounds.top + 220,
+        )
+      })
+    }
+
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
+
+    expect(renderResult.getEditorState().editor.selectedViews).toEqual([desiredPath])
+  })
+
+  it('Seven double clicks will focus a generated Card and select the Button inside', async () => {
+    // prettier-ignore
+    const desiredPath = EP.fromString(
+      'sb' +                 // Skipped as it's the storyboard
+      '/scene-CardList' +    // Skipped because we skip over Scenes
+      '/CardList-instance' + // <- First double click
+      ':CardList-Col' +      // <- Third double click
+      '/CardList-Card~~~1' + // <- Fourth *and* Fifth double click, as the Fifth is required to focus it
+      ':Card-Root' +         // <- Sixth double click
+      '/Card-Row-Buttons' +  // <- Seventh double click
+      '/Card-Button-3',      // <- Eight double click
+    )
+
+    const renderResult = await renderTestEditorWithCode(
+      TestProjectAlpineClimbWithFragments,
+      'await-first-dom-report',
+    )
+
+    const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
+
+    const cardSceneRoot = renderResult.renderedDOM.getByTestId('generated-card-1')
+    const cardSceneRootBounds = cardSceneRoot.getBoundingClientRect()
+
+    const fireDoubleClickEvents = createDoubleClicker()
+
+    const doubleClick = async () => {
+      await act(async () => {
+        fireDoubleClickEvents(
+          canvasControlsLayer,
+          cardSceneRootBounds.left + 130,
+          cardSceneRootBounds.top + 220,
+        )
+      })
+    }
+
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
+    await doubleClick()
```

Comment:
The `desiredPath` description says 8 double clicks and I see only 7 here

Reply:
That is just a bug in the comments on the path, I have just fixed it.
Note: we need one less click than with fragments than with divs, because fragment support is not included in this PR. After fragment support is done we can increase this back to 8